module_intros/diffraction.StaticStructureFactor.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    plt.show()
</pre><p>You'll need to commit this notebook with the plots, since that's what renders in our documentation. Click "Restart and Run All" in Jupyter, then save the file before committing to git.</p><p> </p><p>Example with plots: https://freud.readthedocs.io/en/latest/gettingstarted/examples/module_intros/density.RDF-AccumulateFluid.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>